### DISCUSSION TITLE
Boundary of a phenomenon and its fidelity

### DISCUSSION THREAD

**Discussion by blchoy** [wmo-im/iwxxm#276] [Ideas]
When we started to use WxObject to describe a phenomenon, in case a boundary is used for identification, we will come across issues with regard to the fidelity of the boundary:

![image](https://user-images.githubusercontent.com/30064374/139807617-bce724d7-2eac-49cb-ad4f-4a1a399409ca.png)

It is a general consensus that the number of points involved relates to how well the shape of the phenomenon is described, and the precision of encountering it at its vicinity.

Question is, do we need to include metadata to indicate the fidelity of the boundary?  What should it be (e.g. maximum/nominal/minimal distance of the boundary from the edge of the phenomenon, or otherwise)?

**Comment by moryakovdv** [comment]
Hello.
one more metadata entry comes with little overhead of data processing. 
Nevertheless I would suggest including at least default(nominal in your terms) distance that can be calculated with, for example, severity of the phenomenon.
Just as idea.

**Reply by blchoy** [reply]
Thanks Dmitry.  The metadata I am mentioned is intended to resolve the following issue:

The "tightness" of the boundary determines how much stuff of a different level of a property (e.g. moderate turbulence in a severe turbulence phenomenon) that is included within a boundary.  If the boundary is tight, we expect to have an undulating shape like:

![image](https://user-images.githubusercontent.com/30064374/139814277-01b6ea71-4d76-4606-bf17-e5e406f6faf2.png)

and if the boundary is less tight the shape will be much smoother but there is a greater chance of having stuff of a different level of property:

![image](https://user-images.githubusercontent.com/30064374/139814562-e8fe3aec-277f-4b7d-8f6b-67105f8b5cc3.png)

Now an aircraft flying into a boundary of severe turbulence with a less tight boundary may have the impression that the bounded region of severe turbulence is incorrect because of the greater chance of seeing moderate turbulence.  This is undesirable as in this case performance perception is affected by fidelity of the boundary, but of course we cannot indefinitely raise the resolution of the boundary.

Yes distance to the intended property is something we could do.  Density or probability of occurrence/encountering could be some other thing we can consider.

Glad to hear from other perspectives.  :)

**Comment by efucile** [comment]
Weather phenomena are not as Country boundaries or doughnuts in the sense that they are not precise in their nature. We don't want the fidelity to be confused with the confidence of the localization of the phenomenon.

**Comment by moryakovdv** [comment]
In this case shouldn't we talk about boundaries degradation?

**Comment by blchoy** [comment]
I think Enrico has a point.  But I think for practical purpose the relative sizes of the phenomenon and the user matters.  Take the 7 points in a SIGMET message as an example.  If you are using the 7 points to describe a 10km thunderstorm it should be fine for a flight.  If, however, you are using 7 points to describe space weather phenomena as large as a continent, it would be too coarse.

**Comment by blchoy** [comment]
By the way, in case of TAF the phenomena mentioned has at least a 50% chance of occurrence (or 40% for PROB40 and 30% for PROB30), but there is no description for other products including model outputs.  Correct me if I am wrong.

**Comment by blchoy** [comment]
Our modelling expert indicated that for phenomena with stronger spatial correlation (e.g. volcanic ash concentration) the boundary fidelity effect may be obvious, but for phenomena like icing and turbulence it would be hard to discriminate the boundary effect from the nature of the phenomena.
